src/adagrams.js

```diff
@@ -1,15 +1,85 @@
+import { letterPool, scoreChart } from "./letters";
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  let letterOptions = [];
+
+  for (const [letter, frequency] of Object.entries(letterPool)) {
+    for (let i = 0; i < frequency; i++) {
+      letterOptions.push(letter);
+    }
+  }
+
+  let userLetters = []; //This was origianlly done with a deepcopy method in python
```

Comment:
You could also use the spread operator here like
```javascript
  let letterPoolCopy = [...letterPool];
```
the spread operator is something you see most especially with React. Here is a link to an article that shows all types of ways to make a copy in JS. Number 8 shows how to make a deep copy https://www.freecodecamp.org/news/how-to-clone-an-array-in-javascript-1d3183468f6a/